### PR TITLE
#69 add job name in queue list

### PR DESCRIPTION
**Pull request by MaximeMaillet**

### CODE REVIEW

**Comment by eeVoskos** [comment]
I was about to fork the project and add a job name in the UI, when I saw this PR! +1 for the attempt, although I'd change the code a bit.

You need to check if a `job.name` exists, as it is not a standard in `bull`. Or, even better, I'd add the `name` field in the job's data, not the job object itself.

So, I'd change the `h4` row to something like this:
```
<h4 class="header-collapse"><span class="label label-default">{{ this.id }}</span>
  {{#if this.data.name}}
    <span style="margin-left: 8px">{{ this.data.name }}</span>
  {{/if}}
</h4>
```

**Comment by eeVoskos** [comment]
@randallm @bradvogel would you be kind enough to accept this PR?!

File: src/server/views/dashboard/templates/queueJobsByState.hbs [hunk 1]
```diff
@@ -91,7 +91,14 @@
             </div>
 
             <a role="button" data-toggle="collapse" href="#collapse{{encodeIdAttr this.id}}">
-              <h4 class="header-collapse">{{this.id }}</h4>
+              <h4 class="header-collapse">
+                <span class="label label-default">{{ this.id }}</span>
+                {{#if this.data.name}}
+                  <span style="margin-left: 8px">{{ this.data.name }}</span>
+                {{else if this.name}}
+                    <span style="margin-left: 8px">{{ this.name }}</span>
```
Comment (skeggse) [inline src/server/views/dashboard/templates/queueJobsByState.hbs]: The indentation here is slightly off.

**Comment by skeggse** [comment]
Other than that one nit, this looks good to me. Works as advertised.

image

**Comment by MaximeMaillet** [comment]
@skeggse Go ;)
Thanks

**Comment by skeggse** [comment]
released in v2.2.2

**Pull request merged**